test/charms/main/lib/charm.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
```

Comment:
The filename might be slightly more clear as charms/test_main/... or something like this.. it's reading like "the main charm" now.

Reply:
OK, I'll change it.